tox.ini

```diff
@@ -9,11 +9,10 @@ setenv =
     VIRTUALENV_NO_DOWNLOAD=1
 
 [testenv:lint-shell]
-deps =
-    bashate==0.6.0
+whitelist_externals += shellcheck
```

Comment:
Does tox complain without that? The inherited bash should be enough since it's the only one we call directly.

Reply:
Tox is ok with that, but you're right it's not necessary, I think.